molgenis-integration-tests/src/test/java/org/molgenis/integrationtest/data/abstracts/query/AbstractSearchQueryIT.java

```diff
@@ -68,56 +75,68 @@ protected void testDate() throws ParseException
 	@Override
 	protected void testDateTime() throws ParseException
 	{
-		// FIXME
-		// Query<Entity> query = new QueryImpl<>().search(BIRTH_TIME, "1976-06-07 07:07:07");
-		// Set<Entity> resultSet = Sets.newHashSet(person2);
-		// assertTrue(resultSet.contains(personsRepository.findOne(query)));
-		// assertEquals(personsRepository.findAll(query).collect(toSet()), resultSet);
-		// assertEquals(personsRepository.count(query), resultSet.size());
+		// FIXME Value generated just like the ElasticSearch QueryGeneratorTest.generateOneQueryRuleSearchOneFieldDateTime
+		// FIXME but still does not work
+//		String value = getDateFormat().parse("1976-06-07T07:07:07+0100").toString();
+//
+//	 	Query<Entity> query = new QueryImpl<>().search(BIRTH_TIME, value);
+//	 	Set<Entity> resultSet = Sets.newHashSet(person2);
+// 		assertTrue(resultSet.contains(personsRepository.findOne(query)));
+//	 	assertEquals(personsRepository.findAll(query).collect(toSet()), resultSet);
+//	 	assertEquals(personsRepository.count(query), resultSet.size());
 	}
 
 	@Override
 	protected void testBool()
 	{
-		// UNSUPPORTED
+		try
+		{
+			String value = "true";
+			Query<Entity> query = new QueryImpl<Entity>().search(ACTIVE, value);
+			assertTrue(newHashSet(person2).contains(personsRepository.findOne(query)));
```

Comment:
```assertTrue``` isn't necessary here because you expect the exception to be thrown when you call ```.findOne()```